### ISSUE TITLE
crashes on start 

### ISSUE THREAD

**Issue by pronoob106** [Pan4ur/ThunderHack-Recode#121]
when starting minecraft it crashes before minecraft loads
when i have the mod in the mods folder when i try to launch the game it crashes

**Comment by Pan4ur** [comment]
send crashlog

**Comment by pronoob106** [comment]
okay 
[crash-2023-10-23_11.31.12-client.txt](https://github.com/Pan4ur/ThunderHack-Recode/files/13073487/crash-2023-10-23_11.31.12-client.txt)

**Comment by Pan4ur** [comment]
from https://github.com/Pan4ur/ThunderHack-Recode/releases ?

**Comment by pronoob106** [comment]
yes i got it from that one when i got it, it had said that it was uploaded 3hours before

**Comment by Pan4ur** [comment]
According to the log it does not look like the latest version, make sure that the mods folder is correct and it does not contain 2 thunderhacks, and download again https://github.com/Pan4ur/ThunderHack-Recode/releases/download/latest/thunderhack-1.3.jar

**Comment by pronoob106** [comment]
oh wow thank you i don't really know why it wasn't working but thank you

**Comment by pronoob106** [comment]
also
i have a question

**Comment by Pan4ur** [comment]
?

**Comment by pronoob106** [comment]
can you possibly be able to remove some of the things that make the game lag

**Comment by Pan4ur** [comment]
If you mean visual modules, you can disable them, I don't really know how to make them faster yet

**Comment by Pan4ur** [comment]
No problem
